### PR TITLE
Weather

### PR DESCRIPTION
**Pull request by sakshi006**
added weather component and landing page which will be visible to user once name is entered.
Screenshot (751)

### CODE REVIEW

**Comment by ashifkhn** [comment]
Good to go.

File: src/Pages/Weather/Weather.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            const weatherInfo = res.data
+            setWeather({
+                city: weatherInfo.name,
+                degrees: Math.round(weatherInfo.main.temp - 273.15),
+                weatherIcon: weatherInfo.weather[0].icon,
+                humidity: weatherInfo.main.humidity,
+                description: weatherInfo.weather[0].description,
+            });
+        } catch (err) {
+            alert(err)
```
Comment (krgarima) [inline src/Pages/Weather/Weather.js:36]: You can add Toast.error to show error

File: src/Pages/Weather/Weather.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    };
+    const error = (err) => {
+        weatherAPICall();
+    };
+    const getGeoLocation = () => {
+        navigator.geolocation.getCurrentPosition(success, error);
+    };
+    useEffect(() => {
+        getGeoLocation();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (krgarima) [inline src/Pages/Weather/Weather.js:51]: remove commented codes if not needed

**Review by krgarima — APPROVED** [review]
App is working nicely. Just some minor suggestions to follow.

**Pull request merged**